builder/pipeline.cc

```diff
@@ -680,7 +680,15 @@ class pipeline_builder {
       // And make the actual loop.
       expr loop_step = sanitizer_.mutate(loop.step);
       interval_expr loop_bounds = get_loop_bounds(base_f, loop);
-      body = loop::make(loop.sym(), loop.max_workers, loop_bounds, loop_step, body);
+      // Make sure that a loop variable is unique.
+      std::string loop_var_name = "<";
+      if (!base_f->attrs().name.empty()) {
```

Comment:
I don't think this is a good name, because it isn't necessarily unique. Clients use things like "add" which is used multiple times in the same pipeline.

Maybe we should use the following logic:
- If there is only one output buffer, use that as the prefix.
- If there is more than one output buffer, just use what you had originally (`insert_unique(ctx.name(loop.sym()))`).

Reply:
Done.